docs/HACKING.md

```diff
@@ -117,6 +117,40 @@ hack/run-ci-e2e-test.sh -s -m 2 -c 1
 ```
 Please note that you do not need to run `hack/olm.sh run` before `hack/run-ci-e2e-test.sh`.
 
+### Running e2e tests on platform-agnostic infrastructure
+
+To run the WMCO' e2e tests on a bare metal or other platform-agnostic infrastructure (platform=none), where there
+is no cloud provider specification, the desired number of Windows instances must be available beforehand, and you
+need provide the instance(s) information to the e2e test suite using an environment variable.
```

Comment:
`need to provide the instance(s) information to the e2e test suite using the WINDOWS_INSTANCES_DATA environment variable`